e2e/mobile/specs/swap/otherTestCases/swapUserRefusesTransaction.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { runUserRefusesTransactionTest } from "./swap.other";
+
+const userRefusesTransactionSwapTestConfig = {
+  fromAccount: Account.ETH_1,
+  toAccount: Account.SOL_1,
+  tmsLinks: ["B2CQA-2212"],
+};
+
+runUserRefusesTransactionTest(
+  userRefusesTransactionSwapTestConfig.fromAccount,
+  userRefusesTransactionSwapTestConfig.toAccount,
+  userRefusesTransactionSwapTestConfig.tmsLinks,
+  ["@NanoSP", "@LNS", "@NanoX"],
+);
```

Comment:
maybe it's better like that to mach the tmsLinks and the accounts

```suggestion
const userRefusesTransactionSwapTestConfig = {
  fromAccount: Account.ETH_1,
  toAccount: Account.SOL_1,
  tmsLinks: ["B2CQA-2212"],
  tags: ["@NanoSP", "@LNS", "@NanoX"]
};

runUserRefusesTransactionTest(
  userRefusesTransactionSwapTestConfig.fromAccount,
  userRefusesTransactionSwapTestConfig.toAccount,
  userRefusesTransactionSwapTestConfig.tmsLinks,
  userRefusesTransactionSwapTestConfig.tags,
);
```